docs/source/architectural_design.rst

```diff
@@ -0,0 +1,54 @@
+Architectural design
+====================
+
+Overview
+--------
+
+Given a configuration file (Data Resource Schema) VIA a REST API the Data Resource Generator (DRG) will dynamically generate:
+
+#. Database tables with relationships
+    * SQLAlchemy Object Relation Mappings (ORM)
+    * Generate and run database migrations
+#. A RESTful API
+    * Running Flask application
+    * Open API Specification 3.0
+
+The application is built with Python and uses Flask.
+
+We leverage two open data specifications, Table Schema and Swagger. These are core to the project. By embedding table schema descriptors and a swagger specification along with some metadata you have a Data Resource Schema.
+
+The Data Resource Schema is sent embedded inside a Data Resource Generation Payload to the DRG generation route. This triggers the generation of Data Resources.
+
+The application will stand up as a Flask API. This API has a generation route that holds a reference to the Flask object.
+
+Upon a Data Resource Generation Payload being sent, the application then generates:
+
+* relevant ORM models based on your table schema document
+* relevant REST routes based on your swagger document
+
+Application Paradigm
+--------------------
+
+"Declarative setup - imperative changes."
```

Comment:
Is this a quote from somewhere? If yes, provide a citation. If not, then remove the quotes and use italic font for emphasis.